modules/kubernetes-addons/variables.tf

```
@@ -1197,6 +1197,19 @@ variable "airflow_helm_config" {
   default     = {}
 }
 
+#-----------CloudNative PG (Postresql) ADDON-------------
+variable "enable_cnpg" {
```

Carry chart name throughout naming for consistency:

```suggestion
variable "enable_cloudnative_pg" {
```